geonotebook/kernel.py

```diff
@@ -320,12 +326,21 @@ def serialize(self):
         return ret
 
     def rpc_error(self, error):
-        self.log.error(
-            "JSONRPCError (%s): %s" % (error['code'], error['message'])
-        )
+        try:
+            self.log.error(
+                "JSONRPCError (%s): %s" % (error['code'], error['message'])
+            )
+        except Exception:
+            self.log.error(
+                "JSONRPCError: Malformed error message: {}".format(error)
+            )
```

Comment:
It looks like <https://pypi.python.org/pypi/flake8-pep3101> enforces the `''.format` style.  I don't know of any others.